### PR TITLE
Manual use existing site

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #563

@NyakudyaA, please note that the latest documentation is available in this branch. You can directly merge this branch, as it already incorporates the content from PR #538 and #545.

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 32 sec

File: docs/src/user/manual/map/map.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+![Available References](../img/landing-page-13.png)
+
+1. **Ground Truth**: GroundTruth provides a range of consulting services and equipment beneficial to water resource management projects.
+
+    - GroundTruth has provided input into and coordinated large-scale, multi-national projects as well as small-scale, local projects. Through these projects, we have fostered a large and diverse client base and gained recognition for our work through several awards. 
+
+    ![Ground Truth](../img/landing-page-14.png)
+
+2. **United Nations Children's Fund (UNICEF)**: UNICEF works in over 190 countries and territories to save children's lives, defend their rights, and help them fulfill their potential, from early childhood through adolescence. And we never give up.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/map.md]:
```suggestion
2. **United Nations Children's Fund (UNICEF)**: UNICEF works in over 190 countries and territories to save children's lives, defend their rights, and help them fulfil their potential, from early childhood through adolescence. And we never give up.
```

File: docs/src/user/manual/map/map.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+1. **Home**: Navigate to the home page by clicking on the `Home` link.
+
+2. **HowTo**: Navigate to the how-to page by clicking on the `HowTo` link. For detailed how-to instructions, refer to the [how-to page documentation](../how-to/how-to.md).
+
+3. **Map**: Navigate to the map page by clicking on the `Map` link.
+
+4. **GitHub**: miniSASS is an Open Source Project. Explore and contribute to the source code on GitHub. Click on the GitHub link to get the source code.
+
+
+# Map Page After Login User Manual
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/map.md]:
```suggestion
## Map Page After Login User Manual
```

File: docs/src/user/manual/map/map.md [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+
+    **See Details:** Users can access the details of the uploaded miniSASS observation by clicking on the crab icon.
+
+    ![miniSASS Observation Details](./img/map-10.png)
+
+    1 **Exclamation Mark:** The exclamation mark on the crab indicates that the record has not been validated yet.
+
+    ![Exclamation Mark](./img/map-13.png)
+
+## Available References
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/map.md]:
```suggestion
### Available References
```

File: docs/src/user/manual/map/map.md [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+
+![Available References](../img/landing-page-13.png)
+
+1. **Ground Truth**: GroundTruth provides a range of consulting services and equipment beneficial to water resource management projects.
+
+    - GroundTruth has provided input into and coordinated large-scale, multi-national projects as well as small-scale, local projects. Through these projects, we have fostered a large and diverse client base and gained recognition for our work through several awards. 
+
+    ![Ground Truth](../img/landing-page-14.png)
+
+2. **United Nations Children's Fund (UNICEF)**: UNICEF works in over 190 countries and territories to save children's lives, defend their rights, and help them fulfill their potential, from early childhood through adolescence. And we never give up.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/map.md]:
```suggestion
2. **United Nations Children's Fund (UNICEF)**: UNICEF works in over 190 countries and territories to save children's lives, defend their rights, and help them fulfil their potential, from early childhood through adolescence. And we never give up.
```

File: docs/src/user/manual/map/map.md [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+
+7. **WESSA (Wildlife and Environment Society of South Africa):** WESSA (the Wildlife and Environment Society of South Africa) is a South African environmental organisation that aims to initiate and support high-impact environmental and conservation projects to promote participation in caring for the Earth. For over 90 years we have proactively engaged with the challenges and opportunities presented by our country’s unique natural heritage and the social and economic systems that depend on it.
+
+    ![WESSA!](../how-to/img/how-to-13.png)
+
+8. **Kartoza:** Make geospatial data and technology work for you by partnering with us for training, development and maintenance of GIS systems. Kartoza is a South Africa-based Free and Open Source GIS (FOSSGIS) service provider. We use GIS software to solve complex location-related problems for individuals, businesses and governments around the world. Our team develops software using FOSSGIS to give you the freedom to share and modify your GIS as your needs grow and change.
+
+    ![Kartoza](../how-to/img/how-to-14.png)
+
+## Footer Options
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/map/map.md]:
```suggestion
### Footer Options
```

File: docs/src/user/manual/upload/add-records.md [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+
+Welcome to the Add Record Form User Manual. This guide will walk you through the process of adding a new record using our user-friendly form. Follow the steps below to efficiently input data and contribute valuable information to our system.
+
+![Add Record](./img/add-record-1.png)
+
+1. **Site Details:** Hover over the `Information Icon` next to `Site Details` to view additional information about the section of the form.
+
+2. **Create New Site:** Choose the `Create New Property` option to initiate the process. This option allows the user to add a new property to the site.
+
+    ### Site Details Section
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]: Headings shouldn't be indented in markdown
```suggestion
    **Site Details Section**
```

File: docs/src/user/manual/upload/add-records.md [hunk 7]
```diff
@@ -0,0 +1,213 @@
...
+
+    ![Create a new site, site details section](./img/add-record-2.png)
+
+    1. **Upload Site Images Button:** Use the `Upload Site Images` button to attach visual representations of the property. This helps to provide a comprehensive view and enhance the overall data quality. When the user clicks on the upload site image button a popup will shown for uploading the image of the property.
+
+        #### Upload Image
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
        **Upload Image**
```

File: docs/src/user/manual/upload/add-records.md [hunk 8]
```diff
@@ -0,0 +1,213 @@
...
+
+        1. **River Name:** Enter the name of the river associated with the property.
+
+        2. **Site Name:** Provide a name for the property.
+
+        3. **Site Description:** Add a brief description of the property, highlighting any relevant details, such as its location relative to nearby industries or landmarks.
+
+        4. **River Category:** Hover over the `Information Icon` next to `River Category` to access additional information about the available categories in the dropdown. Select the appropriate category that best represents the river.
+    
+    ### Site Location section
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
    **Site Location section**
```

File: docs/src/user/manual/upload/add-records.md [hunk 9]
```diff
@@ -0,0 +1,213 @@
...
+
+        ![Date Picker](./img/add-record-11.png)
+
+        b. **Collector's Name:** Enter the name of the person collecting the observation data.
+
+        c. **Notes:** Provide any relevant notes or additional information related to the observation.
+    
+    4. **Measurements Section:**
+
+        a. **Water Clarity (cm):** Enter the water clarity measurement in centimeters.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
        a. **Water Clarity (cm):** Enter the water clarity measurement in centimetres.
```

File: docs/src/user/manual/upload/add-records.md [hunk 10]
```diff
@@ -0,0 +1,213 @@
...
+
+        b. **Water Temperature (°C):** Input the water temperature in degrees Celsius.
+
+        c. **pH:** Enter the pH level of the water.
+
+        d. **Dissolved Oxygen:** Choose the dissolved oxygen measurement unit from the dropdown options:
+
+            mg/l (milligrams per liter)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
            mg/l (milligrams per litre)
```

File: docs/src/user/manual/upload/add-records.md [hunk 11]
```diff
@@ -0,0 +1,213 @@
...
+            Unknown
+
+        If the user inserts more than `20` in the `Dissolved Oxygen` field an information message will display.
+
+        ![Information message](./img/add-record-17.png)
+
+        e. **Electrical Conductivity:** Enter the electrical conductivity measurement and select the unit from the dropdown options:
+
+            S/m (Siemens per meter)
+            µS/cm (microsiemens per centimeter)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
            µS/cm (microsiemens per centimetre)
```

File: docs/src/user/manual/upload/add-records.md [hunk 12]
```diff
@@ -0,0 +1,213 @@
...
+
+    5. **Next Button:** To proceed to the next step in adding a record, click on the `Next` button. Upon clicking `Next`, the score form will open, allowing the user to choose the group and upload an image associated with the record. It's important to note that the date field must be filled in before saving the score form.
+
+        ![Score](./img/add-record-12.png)
+    
+    - For detailed instructions on adding a score, refer to the [score documentation](../upload/score.md).
+
+3. **Use Existing Site:** Choose the `Use Existing Site` option to initiate the process. This option allows the user to use the existing property on the site.
+
+    ### Site Details Section
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
    **Site Details Section**
```

File: docs/src/user/manual/upload/add-records.md [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+
+    ![Use existing site, site details section](./img/add-record-13.png)
+
+    1. **Upload Site Images Button:** Use the `Upload Site Images` button to attach visual representations of the property. This helps to provide a comprehensive view and enhance the overall data quality. When the user clicks on the upload site image button a popup will shown for uploading the image of the property.
+
+        #### Upload Image
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
        **Upload Image**
```

File: docs/src/user/manual/upload/add-records.md [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+
+        1. **River Name:** The name of the river will automatically populate with the relevant information of the chosen site. The user is unable to update the value of this field.
+
+        2. **Site Name:** The name of the site will automatically populate with the relevant information of the chosen site. The user is unable to update the value of this field.
+
+        3. **Site Description:** The description of the site will automatically populate with the relevant information of the chosen site. The user is unable to update the value of this field.
+
+        4. **River Category:** The category of the site will automatically populate with the relevant information of the chosen site. The user is unable to update the value of this field.
+
+    ### Site Location section
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
    **Site Location section**
```

File: docs/src/user/manual/upload/add-records.md [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+
+        ![Date Picker](./img/add-record-11.png)
+
+        b. **Collector's Name:** Enter the name of the person collecting the observation data.
+
+        c. **Notes:** Provide any relevant notes or additional information related to the observation.
+
+    4. **Measurements Section:**
+
+        a. **Water Clarity (cm):** Enter the water clarity measurement in centimeters.
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
        a. **Water Clarity (cm):** Enter the water clarity measurement in centimetres.
```

File: docs/src/user/manual/upload/add-records.md [hunk 16]
```diff
@@ -0,0 +1,213 @@
...
+
+        b. **Water Temperature (°C):** Input the water temperature in degrees Celsius.
+
+        c. **pH:** Enter the pH level of the water.
+
+        d. **Dissolved Oxygen:** Choose the dissolved oxygen measurement unit from the dropdown options:
+
+            mg/l (milligrams per liter)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
            mg/l (milligrams per litre)
```

File: docs/src/user/manual/upload/add-records.md [hunk 17]
```diff
@@ -0,0 +1,213 @@
...
+            Unknown
+
+        If the user inserts more than `20` in the `Dissolved Oxygen` field an information message will display.
+
+        ![Information message](./img/add-record-17.png)
+
+        e. **Electrical Conductivity:** Enter the electrical conductivity measurement and select the unit from the dropdown options:
+
+            S/m (Siemens per meter)
+            µS/cm (microsiemens per centimeter)
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/upload/add-records.md]:
```suggestion
            µS/cm (microsiemens per centimetre)
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 27 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 23 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 28 sec

**Pull request merged**